wallet/wallet.go

```diff
@@ -1305,13 +1311,13 @@ func (w *Wallet) TxRecord(txSha *wire.ShaHash) (r *txstore.TxRecord, ok bool) {
 	return nil, false
 }
 
-// openWallet opens a wallet from disk.
-func openWallet() (*Wallet, error) {
-	netdir := networkDir(cfg.DataDir, activeNet.Params)
-	dbPath := filepath.Join(netdir, walletDbName)
+// OpenWallet opens a wallet from disk.
+func OpenWallet(config *Config) (*Wallet, error) {
```

Comment:
This can probably be renamed to Open.  So callers would do `w, err := wallet.Open(cfg)`.